### ISSUE TITLE
lib/libunwind.a -> clash between llvm and libunwind ?

### ISSUE THREAD

**Issue by tseemann** [Linuxbrew/homebrew-core#6698]
```
brew ls -v libunwind  | egrep 'libunwind\.(a|so)'

/home/linuxbrew/.linuxbrew/Cellar/libunwind/1.2/lib/libunwind.so
/home/linuxbrew/.linuxbrew/Cellar/libunwind/1.2/lib/libunwind.a
/home/linuxbrew/.linuxbrew/Cellar/libunwind/1.2/lib/libunwind.so.8.0.1
/home/linuxbrew/.linuxbrew/Cellar/libunwind/1.2/lib/libunwind.so.8
```

```
brew ls -v llvm | egrep 'libunwind\.(a|so)'

/home/linuxbrew/.linuxbrew/Cellar/llvm/6.0.0_1/lib/libunwind.so.1
/home/linuxbrew/.linuxbrew/Cellar/llvm/6.0.0_1/lib/libunwind.a
/home/linuxbrew/.linuxbrew/Cellar/llvm/6.0.0_1/lib/libunwind.so.1.0
```

```
brew uses libunwind
gperftools
```

```
brew uses --recursive libunwind
brewsci/science/atram           brewsci/science/trinity         pound
brewsci/science/express         gperftools                      redis-leveldb
brewsci/science/mira            jumanpp
brewsci/science/scram           leveldb
```

```
Downloading https://linuxbrew.bintray.com/bottles/llvm-6.0.0_1.x86_64_linux.bottle.tar.gz
==> Pouring llvm-6.0.0_1.x86_64_linux.bottle.tar.gz
Warning: llvm dependency berkeley-db was built with a different C++ standard
library (libstdc++ from gcc-4.8). This may cause problems at runtime.
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /home/linuxbrew/.linuxbrew
Could not symlink lib/libunwind.a
Target /home/linuxbrew/.linuxbrew/lib/libunwind.a
is a symlink belonging to libunwind. You can unlink it:
  brew unlink libunwind

To force the link and overwrite all conflicting files:
  brew link --overwrite llvm

To list all files that would be deleted:
  brew link --overwrite --dry-run llvm

Possible conflicting files are:
/home/linuxbrew/.linuxbrew/lib/libunwind.a -> /home/linuxbrew/.linuxbrew/Cellar/libunwind/1.2/lib/libunwind.a
==> Summary
  /home/linuxbrew/.linuxbrew/Cellar/llvm/6.0.0_1: 2,442 files, 4.2GB
```

**Comment by iMichka** [comment]
`llvm` should provide `libunwind` as a resource. Let's try to fix this in #7201
If this works, we may want to drop the `libunwind` formula from linuxbrew.

**Comment by iMichka** [comment]
This has been fixed.
